<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenameTerminalSelectParametersDlg</class>
 <widget class="QWidget" name="RenameTerminalSelectParametersDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbNewName">
       <property name="text">
        <string>New Name:</string>
       </property>
       <property name="buddy">
        <cstring>leNewName</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KLineEdit" name="leNewName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lbTerminal">
     <property name="text">
      <string>Old Terminal:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="TextListWidget" name="lwTerminal"/>
   </item>
   <item>
    <widget class="QCheckBox" name="cbIncludeShadow">
     <property name="text">
      <string>Change shadow lexicon</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbIncludeGrammar">
     <property name="text">
      <string>Adapt grammar</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>TextListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">simonuicomponents/textlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
